.github/workflows/main.yml

```diff
@@ -5,11 +5,19 @@ on: [push, pull_request]
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        java-version: [
```

Comment:
I’d prefer to format this on one line. It’s short and more readable that way.

Reply:
i prefer have this on separate lines. It allows to track changes better (like in enums)